packages/core/src/config/environment.ts

```diff
@@ -38,9 +38,11 @@ export class GlintEnvironment {
   ): GlintEnvironment {
     // Glint V1:
     let additionalGlobals = null;
+    let additionalSpecialForms = {};
     if (topLevelGlintConfigObject) {
       if (topLevelGlintConfigObject.additionalGlobals) {
```

Comment:
Should be `topLevelGlintConfigObject.additionalGlobals || topLevelGlintConfigObject.additionalSpecialForms`?

Reply:
Yes it should be, sorry, I'll do a followup commit with a few tests.

Reply:
Hopefully this should do it. https://github.com/typed-ember/glint/releases/tag/v2.0.0-alpha.10-%40glint%2Fcore

Reply:
Thanks! All looks good here.